mw-webapp/src/styles/_variables.scss

```diff
@@ -1,13 +1,26 @@
+//Font family
+$primaryFontFamily: Inter;
+
 // Font size
-$font-size-large: 48px;
+$primaryFontSize: 12px;
+$fontSizeLarge: 48px;
 
 // Colors
 $shadow-color: rgb(211, 211, 211);
-$primary-background: rgb(255, 255, 255);
+$primaryBackgroundColor: rgb(255, 255, 255);
 $transparent-black: rgba(0, 0, 0, 90%);
 $borderBlack: rgb(0, 0, 0);
-$blackBackground: rgb(0, 0, 0);
```

Comment:
If we use everywnere primary? secondary and similar helpers please rename borderBlack and transparent-black too. And delete please "-". In shadow-colow the same.

Reply:
yeah, right) good job

Reply:
Done